src/main/java/io/cdap/e2e/pages/actions/CdfPipelineRunAction.java

```diff
@@ -89,4 +89,18 @@ public static String captureRawLogs() {
     SeleniumDriver.getDriver().switchTo().window(parent);
     return logs;
   }
+
+  public static int getRecordInCount() {
+    String incount = CdfPipelineRunLocators.recordsInCount.getText();
+    return Integer.parseInt(incount);
+  }
+
+  public static int getRecordOutCount() {
+    String outcount = CdfPipelineRunLocators.recordsInCount.getText();
```

Comment:
Wrong locator

Reply:
corrected and mnoved to CdfHelper to avoid unwanted forwarding